ccmlib/resources/docker/sniproxy/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine
```

Comment:
This file is missing from `MANIFEST.in`, so it isn't copied on `setup.py install`.

Reply:
added it